### ISSUE TITLE
undefined local variable or method `doorkeeper_token'  error with new changes.

### ISSUE THREAD

**Issue by mazharoddin** [owen2345/camaleon-cms#476]
Hi @owen2345 ,
I am getting below error with latest changes.
```
undefined local variable or method `doorkeeper_token' for #<CamaleonCms::Admin::Appearances::ThemesController:0x007fbd35e22878>
  # calculate the current user for API
  def cama_calc_api_current_user
    current_site.users_include_admins.find_by_id(doorkeeper_token.resource_owner_id).try(:decorate) if doorkeeper_token
  end
end

/Users/mazhar/.rvm/gems/ruby-2.3.0@rails4.2.7/bundler/gems/camaleon-cms-75a580e00c7c/app/helpers/camaleon_cms/session_helper.rb:144:in `cama_calc_api_current_user'
/Users/mazhar/.rvm/gems/ruby-2.3.0@rails4.2.7/bundler/gems/camaleon-cms-75a580e00c7c/app/helpers/camaleon_cms/session_helper.rb:110:in `cama_current_user'
/Users/mazhar/.rvm/gems/ruby-2.3.0@rails4.2.7/bundler/gems/camaleon-cms-75a580e00c7c/app/controllers/camaleon_cms/camaleon_controller.rb:98:in `cama_site_check_existence'
```

**Comment by p-decoraid** [comment]
Should be fixed by #478

**Comment by mazharoddin** [comment]
Issue got fixed, thanks @p-decoraid.
